### PR TITLE
Complete documentation

### PR DESCRIPTION
**Pull request by Lhorus6**
- Write the page "Observation" (i.e. exploring-observations.md)
- Rewrite the page "Native feeds" (i.e. feeds.md)

### CODE REVIEW

File: docs/usage/exploring-observations.md [hunk 1]
```diff
@@ -1,7 +1,109 @@
-# Manual creations
+# Observations
 
-!!! tip "Under construction"
+When you click on "Observations" in the left-side bar, you access all the "Observations" tabs, visible on the top bar on the left. By default, the user directly access the "Observables" tab, but can navigate to the other tabs as well.
```
Comment (yassine-ouaamou) [inline docs/usage/exploring-observations.md:3]: the user directly accesses

Comment (yassine-ouaamou) [reply docs/usage/exploring-observations.md:3]: I would rather use the verb "land":
By default, the user directly lands on the "Observables" tab

File: docs/usage/exploring-observations.md [hunk 2]
```diff
@@ -1,7 +1,109 @@
...
+- `Artefacts`: In OpenCTI, the `Artefacts` is a particular Observable. It may contain a file, such as a malware sample.
+- `Indicators`: An `Indicator` is a detection object. It is defined by a search pattern, which could be expressed in various formats such as STIX, Sigma, YARA, among others.
+- `Infrastructures`: An `Infrastructure` describes any systems, software services and any associated physical or virtual resources intended to support some purpose (e.g. C2 servers used as part of an attack, devices or servers that are part of defense, database servers targeted by an attack, etc.).
+
+
+## Observables
+
+### General presentation
+
+An Observable is a distinct entity from the Indicator within OpenCTI and represents an immutable object. Observables can encompass a wide range of entities such as IPv4 addresses, domain names, email addresses, and more. Importantly, Observables doesn't inherently imply malicious intent, they can include items like legitimate IP addresses or domains associated with an organization. Additionally, they serve as raw data points without the additional detection context found in Indicators.
```
Comment (yassine-ouaamou) [inline docs/usage/exploring-observations.md:17]: Observables **don't** inherently imply

File: docs/usage/exploring-observations.md [hunk 3]
```diff
@@ -1,7 +1,109 @@
...
+- History: as described [here](overview.md#history-section).
+
+![Indicator overview](assets/indicator_overview.png)
+
+
+## Infrastructures
+
+### General presentation
+
+An Infrastructure refers to a set of resources, tools, systems, or services employed by a threat to conduct their activities. It represents the underlying framework or support system that facilitates malicious operations, such as the command and control (C2) servers in an attack. Notably, like Observables, Infrastructure doesn't inherently imply malicious intent. It can also represent legitimate resources affiliated with an organization (e.g. devices or servers that are part of defense, database servers targeted by an attack, etc.).
```
Comment (yassine-ouaamou) [inline docs/usage/exploring-observations.md:94]: it's either infrastructures **don't** ..
or **an** infrastructure doesn't

File: docs/usage/feeds.md [hunk 4]
```diff
@@ -1,48 +1,73 @@
 # Native feeds
 
+OpenCTI provides versatile mechanisms for sharing data through its built-in feeds, including Live Stream, TAXII collection, and CSV feeds.
```
Comment (yassine-ouaamou) [inline docs/usage/feeds.md:3]: Live **streams**, TAXII **collections**, and CSV feeds.

**Comment by yassine-ouaamou** [comment]
in my opininon, Live streams should be written with lower case "s" for stream. It's similar to TAXII collections

File: docs/usage/feeds.md [hunk 5]
```diff
@@ -1,48 +1,73 @@
...
 
-* create, update and delete events depending on the filters ;
-* caching already created entities in the last 5 minutes ;
-* resolving relationships and dependencies even out of the filters ;
-* they can be public (without authentication).
+In scenarios involving data sharing between two OpenCTI platforms, Live Streams emerge as the preferred mechanism. Consuming OpenCTI data, whether through a [stream connector](../deployment/connectors.md#stream-section) or within another OpenCTI instance, is most efficiently achieved through Live Streams. These streams operate like TAXII collections but are notably enhanced, supporting:
```
Comment (yassine-ouaamou) [inline docs/usage/feeds.md:23]: it feels like the same thing said three times:
- When sharing data between two OpenCTI platforms, Live Streams are the preferred mechanism.
- In scenarios involving data sharing between two OpenCTI platforms, Live Streams emerge as the preferred mechanism.
- Consuming OpenCTI data, whether through a [stream connector](https://docs.opencti.io/latest/deployment/connectors/#stream-section) or within another OpenCTI instance, is most efficiently achieved through Live Streams.

Comment (Lhorus6) [reply docs/usage/feeds.md:23]: Indeed!

**Pull request merged**